MPF.Check/Program.cs

```diff
@@ -88,11 +88,11 @@ public static void Main(string[] args)
 
                 // Finally, attempt to do the output dance
 #if NET40
-                var resultTask = env.VerifyAndSaveDumpOutput(resultProgress, protectionProgress);
+                var resultTask = env.VerifyAndSaveDumpOutput(resultProgress, protectionProgress, null, opts.Seed);
```

Comment:
I'd prefer this way of invoking:
```suggestion
                var resultTask = env.VerifyAndSaveDumpOutput(resultProgress, protectionProgress, seedInfo: opts.Seed);
```